main.js

```diff
@@ -0,0 +1,237 @@
+  { id: 9,
+    description: "Wilson Men's 2015 Ultra Complete Package Golf Set, Standard",
+    brand: "Wilson",
+    price: "$155.51",
+    image: "https://images-na.ssl-images-amazon.com/images/I/517bv3TTfRL._SL1000_.jpg"},
+  { id: 10,
+    description: "Skullcandy Hesh 2 Bluetooth Wireless Headphones with Mic, Black",
+    brand: "Skullcandy",
+    price: "$49.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/7197WSYscIL._SL1500_.jpg"},
+  { id: 11,
+    description: "Samsung 3.5mm Stereo Headset for Galaxy S5, S4, S3, Note - Non-Retail Packaging - White",
+    brand: "Samsung",
+    price: "$4.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/31tZplZhyEL.jpg"},
+  { id: 12,
+    description: "Beats Studio Wireless Over-Ear Headphone - Titanium",
+    brand: "Beats",
+    price: "$219.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/71Hgw5UEEXL._SL1500_.jpg"},
+  { id: 13,
+    description: "Sony MDRXB950BT/B Extra Bass Bluetooth Headphones (Black)",
+    brand: "Sony",
+    price: "$88.00",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61WaUPGzeKL._SL1200_.jpg"},
+  { id: 14,
+    description: "Beats Solo2 Wireless On-Ear Headphone - Black",
+    brand: "Bose",
+    price: "$149.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61b8ZEFmCJL._SL1500_.jpg"},
+  { id: 15,
+    description: "Bose QuietComfort 35 Wireless Headphones, Noise Cancelling - Black",
+    brand: "Bose",
+    price: "$349.00",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61QwytXOcxL._SL1500_.jpg"},
+  { id: 16,
+    description: "Beats Solo3 Wireless On-Ear Headphone - Rose Gold",
+    brand: "Beats",
+    price: "$249.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/41YByxxpv0L.jpg"},
+  { id: 17,
+    description: "Sennheiser RS120 On-Ear Wireless RF Headphones with Charging Dock",
+    brand: "Sennheiser",
+    price: "$58.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/81NzuWuye9L._SL1500_.jpg"},
+  { id: 18,
+    description: "Bose SoundTrue around-ear headphones II - Apple devices, Charcoal",
+    brand: "Bose",
+    price: "$99.00",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61l0i5ugsfL._SL1500_.jpg"},
+  { id: 19,
+    description: "Amazon Fire TV | Streaming Media Player",
+    brand: "Amazon",
+    price: "$74.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61jPxT1VThL._SL1000_.jpg"},
+  { id: 20,
+    description: "Amazon Echo - Black",
+    brand: "Amazon",
+    price: "$139.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61kJf5cN0gL._SL1000_.jpg"},
+  { id: 21,
+    description: "Roku Streaming Stick (3600R)",
+    brand: "Roku",
+    price: "$39.99",
+    image: "https://images-na.ssl-images-amazon.com/images/I/61bqlBDRRML._SL1500_.jpg"},
+];
+
+var jsText = document.getElementById("text");
```

Comment:
You want variable names from the domain your working in, not the implementation. You also don't want abbreviations in your variable names.